app/services/analytics_events.rb

```diff
@@ -7256,6 +7256,12 @@ def sp_select_email_visited(needs_completion_screen_reason: nil, **extra)
     track_event(:sp_select_email_visited, needs_completion_screen_reason:, **extra)
   end
 
+  # Tracks when user clicks on same tab that user landed on.
+  # @param [String, nil] path that user was on when navigation tab was clicked
+  def tab_navigation_current_page_clicked(path: nil, **_extra)
+    track_event(:tab_navigation_current_page_clicked, path: path)
```

Comment:
Let's pass through `extra` for consistency with other methods.

```suggestion
  def tab_navigation_current_page_clicked(path: nil, **extra)
    track_event(:tab_navigation_current_page_clicked, path:, **extra)
```